src/Sylius/Bundle/AdminBundle/Resources/config/routing/product.yml

```diff
@@ -3,7 +3,7 @@ sylius_admin_product:
         alias: sylius.product
         section: admin
         templates: "@SyliusAdmin\\Crud"
-        except: ['show', 'index', 'update']
```

Comment:
Why did you remove `'update'`?

Reply:
I had a conflict with some PR in this line, and i solved it by remove "update" , i guess it was bad idea :(